insights/2014-04-01/swagger/insightsClient_UsageMetrics.json

```diff
@@ -140,6 +136,14 @@
       "description": "The identifier of the resource.",
       "x-ms-parameter-location": "method",
       "x-ms-skip-url-encoding": true
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version.",
+      "x-ms-parameter-location": "method"
```

Comment:
I see. Then this is fine. However, looking at it for the first time, looks like an anti-pattern. It can be confusing for any one who knows that api-version is usually a client property and see this. Intead of defining it in the global parameters section, please define it inline. Anyways there is only one operation in this spec. So keeping it inline would be better.